src/main/scala/elevate/rise/rules/lowering.scala

```diff
@@ -259,7 +264,7 @@ object lowering {
 
     def apply(e: Rise): RewriteResult[Rise] = e match {
       case reduceResult@App(App(App(ReduceX(), _), _), _) =>
```

Comment:
is the `reduceResult` name needed at all?

Reply:
The name is used here to be able to refer to the type of the entire pattern (i.e. `reduceResult.t`)

Reply:
I mean we can use `e`, correct?